sections/home/ProgrammingTests/Components/TextBlock.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from './TextBlock.module.css';
+
+interface TextBlockProps {
+  title: string;
+  description: string;
+  iconSrc: string;
+}
+
+const TextBlock: React.FC<TextBlockProps> = ({ title, description, iconSrc }) => {
+  return (
+    <div className={styles.textBlock}>
+      <div className={styles.titleContainer}> 
+        <Image src={iconSrc} alt={`${title} icon`} width={32} height={30.35} />
```

Comment:
let's give the image a container that can be resize with custom width so the image be 100% width